bot/utils/decorators.py

```diff
@@ -199,23 +199,38 @@ def predicate(ctx: Context) -> bool:
         kwargs = default_kwargs.copy()
         allow_dms = False
 
-        # Update kwargs based on override
-        if hasattr(ctx.command.callback, "override"):
+        # determine which command's overrides we will use
+        # as we have groups, we want to ensure that group commands inherit from the parent
+        overridden_command: Optional[commands.Command] = None
+        for command in [ctx.command, *ctx.command.parents]:
+            if hasattr(command.callback, "override"):
+                overridden_command = command
+                break
+        if overridden_command is not None:
+            log.debug(f'Command {overridden_command} has overrides')
+            if overridden_command is not ctx.command:
+                log.debug(
+                    f"Command '{ctx.command.qualified_name}' inherited overrides "
+                    "from parent command '{overridden_command.qualified_name}'"
+                )
+
+        # Update kwargs based on override, if one exists
+        if overridden_command and hasattr(overridden_command.callback, "override"):
```

Comment:
We don't need the hasattr check here, right?
```suggestion
        if overridden_command:
```